support-frontend/assets/pages/paper-subscription-checkout/components/thankYou.jsx

```diff
@@ -83,6 +83,7 @@ function ThankYouContent({
         <HeroPicture />
         <HeadingBlock
           overheading="Thank you for supporting our journalism!"
+          overheadingClass="--thankyou"
```

Comment:
as stated above this would become `component-heading-block__overheading--thankyou`